modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -23,14 +23,15 @@ The following Technology Preview features are enabled by this feature set:
 ** CSI automatic migration. Enables automatic migration for supported in-tree volume plugins to their equivalent Container Storage Interface (CSI) drivers. Supported for:
 *** Azure File (`CSIMigrationAzureFile`)
 *** VMware vSphere (`CSIMigrationvSphere`)
-** Shared Resources CSI Driver and Build CSI Volumes in OpenShift Builds. Enables the Container Storage Interface (CSI) (`CSIDriverSharedResource`).
-** CSI volumes. Enables CSI volume support for the {product-title} build system (`BuildCSIVolumes`).
-** Swap memory on nodes. Enables swap memory use for {product-title} workloads on a per-node basis (`NodeSwap`).
-** cgroups v2. Enables cgroup v2, the next version of the Linux cgroup API (`CGroupsV2`).
-** crun. Enables the crun container runtime (`Crun`).
-** Insights Operator. Enables the Insights Operator, which gathers {product-title} configuration data and sends it to Red Hat (`InsightsConfigAPI`).
+** Shared Resources CSI Driver and Build CSI Volumes in OpenShift Builds. Enables the Container Storage Interface (CSI). (`CSIDriverSharedResource`)
+** CSI volumes. Enables CSI volume support for the {product-title} build system. (`BuildCSIVolumes`)
```

Comment:
Not in this PR's scope, but I would find a definition list to be more readable than the current format. For example:

```
* CSI volumes. Enables CSI volume support for the OpenShift Container Platform build system. (`BuildCSIVolumes`)
```
could change to
```
`BuildCSIVolumes`:: Enables CSI volume support for the OpenShift Container Platform build system. 
```

Reply:
100% agreed, but I'm not going to tackle that with this PR since I'm just adding to the existing list and don't otherwise work with this content